### PR TITLE
VIH-7337 New DI to correctly load the configuration

### PR DESCRIPTION
**Pull request by levibutler**
### JIRA link (if applicable) ###
VIH-7337

### Change description ###
New DI to correctly load the configuration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: BookingQueueSubscriber/BookingQueueSubscriber.Common/Configuration/ConfigLoader.cs [hunk 1]
```diff
@@ -13,13 +16,40 @@ public ConfigLoader()
...
                 .AddAksKeyVaultSecretProvider(vhInfraCore)
-                .AddAksKeyVaultSecretProvider(vhBookingQueueSubscriber);
+                .AddAksKeyVaultSecretProvider(vhBookingQueueSubscriber)
+                .AddEnvironmentVariables();
 
             Configuration = configRootBuilder.Build();
         }
     }
+
+    public static class ConfigurationBuilderExtensions
```
Comment (levibutler) [inline BookingQueueSubscriber/BookingQueueSubscriber.Common/Configuration/ConfigLoader.cs]: remove file

File: BookingQueueSubscriber/BookingQueueSubscriber/BookingQueueSubscriber.csproj [hunk 2]
```diff
@@ -8,6 +8,9 @@
     <ProjectGuid>0F855588-E955-4A37-8FFF-49369CDFF7FC</ProjectGuid>
     <UserSecretsId>F6705640-D918-4180-B98A-BAB7ADAA4817</UserSecretsId>
   </PropertyGroup>
+  <ItemGroup>
```
Comment (levibutler) [inline BookingQueueSubscriber/BookingQueueSubscriber/BookingQueueSubscriber.csproj]: cleanup

File: BookingQueueSubscriber/BookingQueueSubscriber/Startup.cs [hunk 3]
```diff
@@ -54,32 +83,14 @@ public static void ConfigureServices(IServiceCollection services)
                     .AddTypedClient(httpClient =>
                     {
                         var client = VideoApiClient.GetClient(httpClient);
-                        client.BaseUrl = hearingServicesConfiguration.VideoApiUrl;
+                        client.BaseUrl = serviceConfiguration.VideoApiUrl;
                         client.ReadResponseAsString = true;
```
Comment (shaed-parkar) [inline BookingQueueSubscriber/BookingQueueSubscriber/Startup.cs:87]: no longer necessary to set `ReadResponseAsString` to true

**Review by shaed-parkar — DISMISSED** [review]
Nice work! LGTM

**Review by shaed-parkar — DISMISSED** [review]

**Review by levibutler — COMMENTED** [review]
LGTM

**Review by shaed-parkar — APPROVED** [review]

**Pull request merged**